src/libs/NumberUtils.js

```diff
@@ -36,23 +36,7 @@ function rand64() {
     return left + middleString + rightString;
 }
 
-/**
```

Comment:
Remove this piece of code that was moved to NumberUtils [here](https://github.com/Expensify/App/pull/10696/files)